docs/advanced/plugins.mdx

```diff
@@ -53,7 +53,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.mdx?$/,
+        test: /\.mdx$/,
```

Comment:
Push our own extension.
Several tools already handle markdown (e.g., parcel), which we shouldn’t overwrite. And MDX is somewhat different, so it’s good to push the different extension